torchqdynamics/solvers/_base.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from typing import List
+
+import numpy as np
+import torch
+
+
+@dataclass
+class BaseSolverOption:
+    H: torch.Tensor
+    jump_ops: List[torch.Tensor]
+    expect_ops: List[torch.Tensor]
+    t_save: np.array
+
+
+@dataclass
+class FixedStepOption(BaseSolverOption):
+    pass
```

Comment:
`dt`for `times`? I don't have an opinion but we have to choose

Reply:
I'm not sure to understand your question. IMO we have `t_save` specifying when we want to have the results (common to all solvers), and a fixed `dt` for the fixed step schemes. It would be interesting to have a fixed step solver `dt` that _depends on time_ according to the user choice (for some use case where the user knows the dynamic is much faster in some time interval than in other). But it's a very specific feature, I don't think we want to care about it now. So I don't see the need of a list of times in this case.